DirectEmployers/MyJobs
# [PD-3274] improve criteria presentation scroll

## What's this PR do?
PR updates the main sidebar navigation on desktop and mobile with the breadcrumbs and navigation

## Screenshots (if appropriate)
![screen shot 2017-06-27 at 3 57 57 pm](https://user-images.githubusercontent.com/22029077/28367280-4391e23a-6c5e-11e7-9256-92d622853858.png)
![screen shot 2017-06-27 at 3 58 24 pm](https://user-images.githubusercontent.com/22029077/28367286-48eeefac-6c5e-11e7-9935-cf02399e9cba.png)


## Where should the reviewer start?
Start inside any of the analytics applications

## How should this be manually tested?
Pull down the repository. Go inside of analytics and look at the sidebar in desktop and mobile and you will see the changes. The screenshots are updated in this PR as well to show what they should look like. Along with that, if you go inside of the myjobs folder and run dkgg watch you will see the test run from editing the main reducer test to include the mobilecrumbs for the sidebar

## Any background context you want to provide?
If you find any commented out code or anything that needs updated please let me know. There were a decent amount of files changed and stuff flipped around so I lost track a bit and went back through but I could have missed something.
